lms/djangoapps/courseware/inline_analytics_utils.py

```diff
@@ -9,8 +9,8 @@
 
 from xmodule.capa_module import CapaModule
 
-ValidResponse = namedtuple('ValidResponse', 'correct_response response_type')  # pylint: disable=C0103
-AnalyticsContextResponse = namedtuple('AnalyticsContextResponse', 'id correct_response response_type message')  # pylint: disable=C0103
+ValidResponse = namedtuple('ValidResponse', 'correct_response response_type choice_name_list')  # pylint: disable=C0103
```

Comment:
What Pylint violation is `C0103`? Since we're already editing these lines, it'd be nice to update them w/ the pylint string instead of the number.